src/index.html

```diff
@@ -13,41 +13,76 @@
     <header class="header">
       <a
         class="logo" 
-        href="http://localhost:8080/">
+        href="http://localhost:8080/"
+      >
         <img src="/images/logo.png" alt="Logo">
       </a>
       <nav class="nav">
-        <ul>
-          <li>
+        <ul class="nav__list">
+          <li class="nav__link-container">
             <a 
-              class="nav_link is-active" 
-              href="http://localhost:8080/">apple
+              class="nav__link is-active" 
+              href="http://localhost:8080/"
+            >
+              apple
             </a>
           </li>
-          <li>
-            <a class="nav_link" href="http://localhost:8080/">samsung</a>
+          <li class="nav__link-container">
+            <a 
+              class="nav__link" 
+              href="http://localhost:8080/"
+            >
+              samsung
+            </a>
           </li>
-          <li>
-            <a class="nav_link" href="http://localhost:8080/">smartphones</a>
+          <li class="nav__link-container">
+            <a 
+              class="nav__link" 
+              href="http://localhost:8080/"
+            >
+              smartphones
+            </a>
           </li>
-          <li>
+          <li class="nav__link-container">
             <a 
               data-qa="hover" 
-              class="nav_link" 
-              href="http://localhost:8080/">laptops &#38 computers
+              class="nav__link" 
+              href="http://localhost:8080/"
+            >
+              laptops &#38 computers
             </a>
           </li>
-          <li>
-            <a class="nav_link" href="http://localhost:8080/">gadgets</a>
+          <li class="nav__link-container">
+            <a 
+              class="nav__link" 
+              href="http://localhost:8080/"
+            >
+              gadgets
+            </a>
           </li>
-          <li>
-            <a class="nav_link" href="http://localhost:8080/">tablets</a>
+          <li class="nav__link-container">
+            <a 
+              class="nav__link" 
+              href="http://localhost:8080/"
+            >
+              tablets
+            </a>
           </li>
-          <li>
-            <a class="nav_link" href="http://localhost:8080/">photo</a>
+          <li class="nav__link-container">
+            <a 
+              class="nav__link" 
+              href="http://localhost:8080/"
+            >
+              photo
+            </a>
           </li>
-          <li class="nav_link_last">
-            <a class="nav_link" href="http://localhost:8080/">video</a>
+          <li class="nav__link-container nav__link-container--last">
```

Comment:
`nav__link-container--last` it isn't good practice, delete this class and use pseudo-classes in styles